jf_agent/git/__init__.py

```diff
@@ -103,6 +103,13 @@ class NormalizedPullRequestComment:
     system_generated: bool = None
 
 
+@dataclass
+class NormalizedPullRequestRepository:
```

Comment:
Why not use [`NormalizedShortRepository`](https://github.com/jellyfish-ai/jf_agent/blob/3752f1a4f888a09d4b6bd18c859f4969c7115849/jf_agent/git/__init__.py#L63-L67)?

Reply:
whoops, missed that, good catch